kod/object/passive/spell/jala/restorat.kod

```diff
@@ -37,6 +37,8 @@ classvars:
    viMana = 5           % Mana is amount used upon inititiation
    viManaDrain = 3      % Drain is amount used every viDrainTime milliseconds
    viDrainTime = 5000   % Drain some mana every viDrainTime milliseconds
+   viSpellPowerIntercept = 40    % Hypothetical zero spell power restorate effect coefficient
+   viSpellPowerSlope = 400       % 1.12 to 1.51 times faster HP regen, linear effect on time per tick
```

Comment:
Please put constants in the "constants" section, not classvars.

Reply:
Sorry for confusion.  Done.  I copied constants formatting from another file.